invenio_oaiserver/search/component_templates/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016-2018 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Module with JSON schemas for an internal ``_oai`` field."""
```

Comment:
The name of the field does not start with '_' anymore
```suggestion
"""Module with JSON schemas for an internal ``oai`` field."""
```